rest/__main__.py

```diff
@@ -17,4 +23,100 @@ def hello():
     return response
 
 
+@app.route('/user', methods=['POST'])
```

Comment:
Put all this routes in different file, we don't want to keep all routes in one file.